packages/cli/src/index.ts

```diff
@@ -145,6 +145,7 @@ export async function build(_options: CliOptions) {
       {
         preflights: options.preflights,
         minify: options.minify,
+        id: postTransform[0].id,
```

Comment:
@Simon-He95 Maybe the id at index 0 is not the correct id and may be filtered out.

Reply:
This is indeed the case, but postTransform has flattened the codes of all files together. Otherwise, you can provide a parameter to determine whether it is a file type. If not, then use the file to generate css and merge it. You what do you think